backend/src/database/migrations/20221208074426-add-should-campaign-should-bcc.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('campaigns', 'should_bcc_to_me', {
+      type: Sequelize.DataTypes.BOOLEAN,
```

Comment:
recommend adding `allowNull: false` due to use-case definition of this field as sequelize default allowNull param is true.